components/Storyblok/SbMegaMenu/SbMegaMenuLinkGroup.tsx

```diff
@@ -30,7 +30,6 @@ export const SbMegaMenuLinkGroup = ({ blok }: SbMegaMenuLinkGroupProps) => {
               <CtaLink
                 sbLink={link}
                 variant="mega-menu-link-lvl2"
-                align="right"
```

Comment:
Remove `align="right"` prop that got added when I copy pasta 🤦🏼‍♀️  This screw up was only visible on Safari where all the mega menu links (links inside panel) were right aligned.